src/nodemanagerbase.cpp

```diff
@@ -181,6 +269,83 @@ UaStatus NodeManagerBase::addNodeAndReference(
 
 	    return s;
 	}
+
+    case OpcUa_NodeClass_Method:
+    {
+    	UA_MethodAttributes attr;
+    	UA_MethodAttributes_init(&attr);
+    	attr.executable = true;
+    	attr.userExecutable = true;
+    	attr.displayName = *displayName.impl();
+    	attr.description = UA_LOCALIZEDTEXT("en_US","the description");
+
+    	MethodHandleUaNode *handle = new MethodHandleUaNode;
```

Comment:
short answer: no. This handle is required for the whole lifetime of server instance, so it won't be a "continuous leak" or something, but you are right, it is kind of wrong. My problem is that I don't see an obvious place where to store it, apart from inventing a std::list with things to free at server exit. And the size of structure to store it would probably equal the size of handles themselves. Any idea?